src/Sentry.Maui/Internal/MauiCustomSessionReplayMaskBinder.cs

```diff
@@ -9,25 +9,40 @@ namespace Sentry.Maui.Internal;
 /// <summary>
 /// Masks or unmasks visual elements for session replay recordings
 /// </summary>
-internal class MauiVisualElementEventsBinder : IMauiElementEventBinder
+internal class MauiCustomSessionReplayMaskBinder : IMauiElementEventBinder
 {
     private readonly SentryMauiOptions _options;
+    private readonly bool _isEnabled;
 
-    public MauiVisualElementEventsBinder(IOptions<SentryMauiOptions> options)
+    public MauiCustomSessionReplayMaskBinder(IOptions<SentryMauiOptions> options)
     {
         _options = options.Value;
+#if __ANDROID__
+        var replayOptions = _options.Native.ExperimentalOptions.SessionReplay;
+        _isEnabled = (replayOptions.SessionSampleRate > 0.0 || replayOptions.OnErrorSampleRate > 0.0)
+            && replayOptions is not { MaskAllImages: true, MaskAllText: true }
```

Comment:
Actually I'm not sure about this condition. I think we can...
- _either_ configure `{ MaskAllImages: true, MaskAllText: true }` and override this with the `sentry:SessionReplay.Mask="Unmask"` attribute 
- _or_ configure `{ MaskAllImages: false, MaskAllText: false }` and override this with the `sentry:SessionReplay.Mask="Mask"` attribute